src/onboarding/steps.ts

```diff
@@ -127,8 +129,8 @@ export function getOnboardingStepValues(screen: Screens, onboardingProps: Onboar
     currentScreen = nextScreen
   }
 
-  const toHomeStep = () => {
-    currentScreen = Screens.WalletHome
+  const finishOnboarding = (nextScreen: Screens) => {
```

Comment:
i wonder if this should log an error or something if nextScreen isn't in END_OF_ONBOARDING_SCREENS

Reply:
we could just make the arg of type of elements in the `END_OF_ONBOARDING_SCREENS` array so we can catch it compile time.